rfcs/text/0000/destination.yml

```diff
@@ -0,0 +1,7 @@
+- name: destination
+  fields:
+    - name: hostname
```

Comment:
With the `source`/`destination`/`client`/`server` field sets, the address value should populate the `.address` field and be duplicated to the appropriate field based on the value:`.ip` for IP addresses, `.domain` for FQDNs or hostnames.

I believe the `.domain` field serves the same function you're proposing here. Or do you have different motivations for proposing this addition?

Reply:
Using the example of _somehost.example.com_ as a fully qualified domain name: 
where some _somehost_ is the hostname 
and _example.com_ is the domain name
See https://docs.microsoft.com/en-us/windows/win32/dns/naming-conventions
There is general confusion when the FQDN can be both .domain as well as host.name.  
The point here is to isolate the hostname (I.E. _somehost_), as well as the domain (._domain_) for a more accurate and reliable representation of the data and for ease of user search.

Reply:
I think this addition can make sense here.